app/models/ipeds_hd.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class IpedsHd < ImportableRecord
-  COLS_USED_IN_INSTITUTION = %i[vet_tuition_policy_url f1sysnam f1syscod].freeze
+  COLS_USED_IN_INSTITUTION = %i[vet_tuition_policy_url f1sysnam f1syscod latitude].freeze
```

Comment:
This is not needed since you are manually adding the column in `institution_builder`